### PR TITLE
feat(pkger): add stack init cmd to influx cli

### PR DESCRIPTION
**Pull request by jsteenb2**
closes: #17235

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: cmd/influx/pkg.go [hunk 1]
```diff
@@ -345,6 +345,74 @@ func (b *cmdPkgBuilder) cmdPkgValidate() *cobra.Command {
 	return cmd
 }
 
+func (b *cmdPkgBuilder) cmdStack() *cobra.Command {
+	cmd := b.newCmd("stack", nil, false)
+	cmd.Short = "Stack command is the container for all stack CRUD"
```
Comment (jademcgough) [inline cmd/influx/pkg.go]: Could just be `"stack management commands"` to be consistent with other descriptions.

File: cmd/influx/pkg.go [hunk 2]
```diff
@@ -345,6 +345,74 @@ func (b *cmdPkgBuilder) cmdPkgValidate() *cobra.Command {
...
+	if err != nil {
+		return err
+	}
+
+	orgID, err := b.org.getID(orgSVC)
+	if err != nil {
+		return err
+	}
+
+	const fakeUserID = 0 // is 0 because user is pulled from token...
```
Comment (jademcgough) [inline cmd/influx/pkg.go]: The userID can be inferred from the token if you use the auth service, but I'm not sure I understand what it's being used for here.

Is it a problem for the userID to be 0 in some cases?

File: cmd/influx/pkg.go [hunk 3]
```diff
@@ -345,6 +345,74 @@ func (b *cmdPkgBuilder) cmdPkgValidate() *cobra.Command {
...
+	return cmd
+}
+
+func (b *cmdPkgBuilder) cmdStackInit() *cobra.Command {
+	cmd := b.newCmd("init", b.stackInitRunEFn, true)
+	cmd.Short = "Initialize a stack"
+
+	cmd.Flags().StringVarP(&b.name, "stack-name", "n", "", "Name given to created stack")
+	cmd.Flags().StringVarP(&b.description, "stack-description", "d", "", "Description given to created stack")
+	cmd.Flags().StringArrayVarP(&b.urls, "package-url", "u", nil, "Package urls to pull associate with new stack")
```
Comment (jademcgough) [inline cmd/influx/pkg.go]: Remove "pull"

**Comment by jsteenb2** [comment]
@jademcgough that's a good question. Say you are boostrapping your app for the first time. You've created your org, user, bucket, and now you want a stack to associate with some packages you want in the platform. That stack can be bootstrapped with this call here. The stack can have associated urls, which apply will automatically pull and run for the user associated with the stack_id provided. 

Does that help illuminate the need for this?

File: cmd/influx/pkg.go (hunk 2 above)
Comment (jsteenb2) [reply cmd/influx/pkg.go]: If the ID is real or not, the user assoicated with the token is what is used when creating resources at this time. That is pretty consistent across the API at this pint.

**Pull request merged**